python/lsst/rubintv/production/rubinTv.py

```diff
@@ -141,6 +144,30 @@ def __init__(self):
         self.bucket = self.client.get_bucket('rubintv_data')
         self.log = _LOG.getChild("googleUploader")
 
+    def uploadHeartbeat(self, channel, flatlinePeriod):
+        filename = "/".join([self.HEARTBEAT_PREFIX, channel]) + ".json"
+
+        currTime = int(time.time())
+        nextExpected = currTime + flatlinePeriod
+
+        heartbeatJsonDict = {
+            "channel": channel,
+            "currTime": currTime,
+            "nextExpected": nextExpected,
+            "errors": {}
+        }
+        heartbeatJson = json.dumps(heartbeatJsonDict)
+
+        blob = self.bucket.blob("/".join([filename]))
+        blob.cache_control = 'no-store'  # must set before upload
+        blob.upload_from_string(heartbeatJson)
+
+        self.log.debug(f'Uploaded heartbeat to channel {channel} with datetime {currTime}')
+
+        blob.reload()
+        if blob.cache_control != 'no-store':
```

Comment:
I realize this isn't supposed to trigger, but is it possible to trigger?  Does `blob.reload()` potentially alter the cache_control property?

Reply:
It certainly shouldn't be able to trigger, no. Given that I've never seen it do so (it was mainly put in as an assert type statement during development while we were trying to get the caches to behave) it could probably safely come out, but we also don't care about speed here at all, and if it were ever to fire I'd quite like the error to be raised, so I think leaving it is probably best on balance.

Reply:
Plus in the not so distant future we'll be moving from GCS to S3, so all of this will have to go at that point.